skel/.vimrc

```diff
@@ -0,0 +1,18 @@
+set number
+set smartcase
+set autoindent
+set nowrap
+set incsearch
+set hlsearch
+set cursorline
+set whichwrap=b,s,<,>,[,]
+set backspace=2
+set ruler
+set laststatus=2
+set noshowcmd
+set ttyfast
+colo elflord
```

Comment:
No point in shortening the commands in vimscript.

Reply:
Yes, I was lazy there :)

This is actually the critical change, see my comment below.